src/shared/Form/Form.tsx

```diff
@@ -0,0 +1,40 @@
+import Modal from '@material-ui/core/Modal';
+import MailIcon from '@material-ui/icons/Mail';
+import React, { useState } from 'react';
+
+import classes from './Form.module.css';
+
+const Form = ({title }: {title: string }): JSX.Element => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const handleOpen = () => {
+    setIsOpen(true);
+  };
+
+  const handleClose = () => {
+    setIsOpen(false);
+  };
+
+  return (
+    <>
+      <MailIcon onClick={handleOpen} style={{fontSize: 26, color: 'white', cursor: 'pointer'}}/>
+      <Modal
+        open={isOpen}
+        onClose={handleClose}
+        aria-labelledby="simple-modal-title"
+        aria-describedby="simple-modal-description"
+      >
+        <>
+          <form className={`${classes.positionModal}`}>
```

Comment:
Why we don't use MUI Inputs? The styles of this inputs are not the same and we have UI conflicts.